recipes/plumpy/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "plumpy" %}
+{% set version = "0.7.9"%}
+{% set sha256 = "a14c20c61c086414895142d22d4ba9b4a5267a743dddfff860d947a5126fdb79" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}-fixed.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-fixed.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python 2.7*
+    - setuptools
+
+  run:
+    - python 2.7*
+    - frozendict
+    - enum34
+    - futures
+
+test:
+  imports:
+    - plum
+
+  commands:
+    - python -m unittest discover
+
+  source_files:
+    - test
+
+about:
+  home: https://bitbucket.org/aiida_team/plum/
```

Comment:
Is the bitbucket the official repo? There's also: 

https://github.com/aiidateam/plumpy

and the repos on Github seem to be more recently updated.